unit-tests/src/test/scala/java/lang/ProcessTest.scala

```diff
@@ -106,7 +106,7 @@ class ProcessTest {
 
   @Test def inputStreamWritesToFile(): Unit = {
     val pb = new ProcessBuilder("echo.sh")
-    pb.environment.put("PATH", resourceDir)
+    pb.environment.put("PATH", (resourceDir :: bashExtraPaths).mkString(":"))
```

Comment:
Sorry for my lack of understanding.  What happens if, Heresy!, the shell is not bash?  Has that case been tested?  Should the name be, say, `shellExtraPaths`

I, for one, have never run the build on anything but bash and am
not sure if that is a requirement outside of `scripted`.

Reply:
This is not related to the shell you are using. This test is launching `echo.sh` and that is a bash script that previously had a fixed `/bin/bash` in the shebang line. All other scripts work with plain `/bin/sh`, this is the only one that requires bash, and it may be in a different path on other platforms. If in future it'll be added a test that requires zsh (I highly doubt) I guess we can change the name later.

Reply:
Can `echo.sh` be changed to use `/bin/sh` rather than these extra changes?

Reply:
Done